components/compliance-service/reporting/relaxting/depth.go

```diff
@@ -160,7 +161,7 @@ func getControlLevelControlSums(hit *elastic.SearchHit) (nodeControlSummary repo
 			}
 		}
 	}
-	return
+	return // nolint:nakeret
```

Comment:
This function and most of the functions in this file appear to drop error information. @rickmarry is looking into it.

Reply:
https://github.com/chef/automate/issues/1763